### PR TITLE
Updated nginx role with web and ssh fw rules

### PR DESCRIPTION
**Pull request by jpolivera**
- Added SSH rules to the Nginx role as discussed
- Relocated authorized hosts to /etc/ansible/access (I guess we will have to change this again later) once we figure out the right location within the ansible-deploy-config-example repo

### CODE REVIEW

File: roles/nginx/tasks/main.yml [hunk 1]
```diff
@@ -106,3 +106,30 @@
...
+  ufw:
+    comment: "Allow HTTPS acccess"
+    from_ip: "{{item}}"
+    port:    "443"
+    rule:    "allow"
+  with_lines: cat /etc/ansible/access/web
+
+- name: Generate SSH Firewall Rules
+
+  ufw:
```
Comment (pm98zz-c) [inline roles/nginx/tasks/main.yml]: SSH access should be split into 2:
- a single rule in the "firewall" module that simply allows access from the provisioning server (avoid ansible locking itself out, basically). I think you should be able to gather this information from environment variable SSH_CLIENT (to be verified)
- the rest in a ssh_server role (or sshd or whatever we want to name it)

File: roles/nginx/tasks/main.yml [hunk 2]
```diff
@@ -106,3 +106,30 @@
...
     state: restarted
+
+- name: Generate HTTP Firewall Rules
+
+  ufw:
+    comment: "Allow HTTP acccess"
+    from_ip: "{{item}}"
+    port:    "80"
+    rule:    "allow"
+  with_lines: cat /etc/ansible/access/web
```
Comment (pm98zz-c) [inline roles/nginx/tasks/main.yml]: This won't work until we have something defined under /etc/ansible/access/web, so won't work for anyone else but us. we probably should add a default variable (that simply is a rule allowing all).
```
nginx.firewall:
```

We can then override this with our own list in the config repo.

File: roles/nginx/tasks/main.yml (hunk 1 above)
Comment (jpolivera) [reply roles/nginx/tasks/main.yml]: - Added the rule with SSH_CLIENT but I'm having a hard time making it work. Please check the `ufw` branch for details.
- Removed SSH config from nginx role.
- Will be creating the `sshd` role next.

Comment (jpolivera) [reply roles/nginx/tasks/main.yml]: MR created for `sshd`

File: roles/nginx/tasks/main.yml (hunk 2 above)
Comment (jpolivera) [reply roles/nginx/tasks/main.yml]: A default rule in the `firewall` role allowing all for Nginx and then restrict here?

Could you show me an example of how to use `group_vars` in the config repo?

**Pull request closed without merging**